btree/btree.c

```diff
@@ -7352,20 +7363,46 @@ static int64_t btree_get_kv_tick(struct m0_sm_op *smop)
 				return bnode_get(&oi->i_nop, tree, &child,
 						 P_NEXTDOWN);
 			} else {
+				if ((lev->l_idx == bnode_count(lev->l_node)) &&
+				    (!oi->i_key_found) &&
+				    (bop->bo_flags & BOF_SLANT)) {
+						bnode_unlock(lev->l_node);
+						return P_SIBLING;
+					}
```

Comment:
Extra tabs in above 3 lines.
Kindly remove them.

Reply:
Done